typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
+  const value: any
```

Comment:
Why do we need this?

Reply:
TypeScript can't import JSON files without this one.